Tests.Integration.Presentation.Web/SystemUsage/ItSystemUsageOverviewReadModelsTest.cs

```diff
@@ -801,13 +812,13 @@ public async Task ReadModels_DependsOnInterfacesNamesAsCsv_Is_Updated_When_Syste
             Console.Out.WriteLine("Read model found");
 
             Assert.Equal("", relationSystemReadModel.IncomingRelatedItSystemUsagesNamesAsCsv);
-            Assert.Empty(relationSystemReadModel.IncomingRelatedItSystemUsages); 
+            Assert.Empty(relationSystemReadModel.IncomingRelatedItSystemUsages);
             Assert.Equal("", relationSystemReadModel.DependsOnInterfacesNamesAsCsv);
             Assert.Empty(relationSystemReadModel.DependsOnInterfaces);
         }
 
         [Fact]
-        public async Task ReadModels_DependsOnInterfacesNamesAsCsv_Is_Updated_When_SystemRelation_Is_Changed()
```

Comment:
Nu vi havde den del med at den ikke opdaterede relations korrekt når vi slettede dem. Skal vi så ikke tilføje en test der tjekker den del?

Reply:
Der var følgende fejl

- [ ] Read models opdateres ikke når selve usage nedlægges
- [ ] Read model opdateres ikke når navnet på en to/from usage opdateres

Dem ser jeg lige på. Den ovenfor medtager den udgående relation men det burde være nemt at tilføje de to ekstra for skelettet er næsten det samme